crates/ruff_server/README.md

```diff
@@ -9,8 +9,31 @@ files in your editor's workspace, and will refresh its in-memory configuration w
 
 We have specific setup instructions depending on your editor. If you don't see your editor on this list and would like a setup guide, please open an issue.
 
-- Visual Studio Code: Install the [Ruff extension from the VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff). The language server used by the extension will be, by default, the one in your actively-installed `ruff` binary. If you don't have `ruff` installed and haven't provided a path to the extension, it comes with a bundled `ruff` version that it will use instead. Since the new Ruff language server has not yet been stabilized, you will need to use the pre-release version of the extension and enable the `Experimental Server` setting.
-- Neovim: See the [Neovim setup guide](docs/setup/NEOVIM.md).
+#### Visual Studio Code
+
+Install the [Ruff extension from the VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff).
+
+As this server is still in beta, you will need to enable the `Native Server` extension setting:
+
+![A screenshot showing an enabled "Native Server" extension setting in the VS Code settings view](assets/nativeServer.png)
+
+You can also set it in your user / workspace JSON settings as follows:
+
+```json
+"ruff.nativeServer": true
```

Comment:
Let's make this valid JSON, to match what we have here: https://github.com/astral-sh/ruff-vscode?tab=readme-ov-file#enabling-the-rust-based-language-server.